changelogs/current.yaml

```diff
@@ -430,7 +430,11 @@ new_features:
 - area: lds
   change: |
     pause SRDS when LDS is updated.
-
+- area: original_dst
+  change: |
+    Filter state is pulled from request context first (if available), then falls back to connection context. Added ability
+    to pick host from dynamic metadata using a key. Same behavior - looks in request context first (if available), falls back
+    to connection context.
```

Comment:
Please ref link to the new field.

/wait

Reply:
Done